### PR TITLE
Generate Multus config file regardless

### PR DESCRIPTION
**Pull request by pliurh**
During deployment, the daemonSet will alway generate the multus config file no matter if file '00-multus.conf' exists or not.

Fixes: #265

### CODE REVIEW

File: images/entrypoint.sh [hunk 1]
```diff
@@ -157,12 +157,10 @@ fi
 
 if [ "$MULTUS_CONF_FILE" == "auto" ]; then
   echo "Generating Multus configuration file ..."
-  MASTER_PLUGIN="$(ls $CNI_CONF_DIR | grep -E '\.conf(list)?$' | head -1)"
```
Comment (booxter) [inline images/entrypoint.sh]: This is not directly related to your patch, but I think there is a bug here, because this line makes an assumption that the order in which ```ls``` lists directory entries is predetermined (and perhaps alphanumeric). I don't think it's a valid assumption, not on every file system type at least.

Comment (pliurh) [reply images/entrypoint.sh]: At least on Linux platform, this assumption is true.
In man of 'ls', it says "List information about the FILEs (the current directory by default).  Sort entries alphabetically if none of -cftuvSUX nor --sort is specified."
